e2e/test/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -21,11 +21,10 @@ describe("scenarios > x-rays", () => {
 
   const XRAY_DATASETS = 11; // enough to load most questions
 
-  it.skip("should not display x-rays if the feature is disabled in admin settings (metabase#26571)", () => {
+  it("should not display x-rays if the feature is disabled in admin settings (metabase#26571)", () => {
     cy.request("PUT", "api/setting/enable-xrays", { value: false });
 
     cy.visit("/");
-    cy.findByText("Metabase tips");
```

Comment:
Was this check creating problems?

Reply:
When a user does not have any recent or popular items we don't show "Metabase tips"